README.md

```diff
@@ -243,6 +243,8 @@ For more information about the governance of the Node.js project, see
 **Evan Lucas** &lt;evanlucas@me.com&gt; (he/him)
 * [fhinkel](https://github.com/fhinkel) -
 **Franziska Hinkelmann** &lt;franziska.hinkelmann@gmail.com&gt; (she/her)
+* [gibfahn](https://github.com/gibfahn) -
```

Comment:
Wouldn't `gibfahn` come after `Fishrock123`?

Reply:
Whoops, yes, thanks for catching that.